raws/coil_arrange/MakeSingleCoilArrangement.py

```diff
@@ -9,59 +9,59 @@
 
 
 Array               = []
-
+radius = 20
```

Comment:
Again, try to avoid hardcoded values. If it is really necessary, make sure you put sufficient comments around it to make other developer to understand it.

Reply:
this change is not necessary.